install.md

```diff
@@ -118,42 +116,134 @@ yum install -y \
 Debian, Ubuntu, and related distributions:
 
 ```bash
-apt-get install -y \
+sudo apt-get install \
   btrfs-tools \
   git \
   golang-go \
   go-md2man \
   iptables \
   libassuan-dev \
+  libc6-dev \
   libdevmapper-dev \
   libglib2.0-dev \
-  libc6-dev \
-  libgpgme11-dev \
+  libgpgme-dev \
   libgpg-error-dev \
+  libostree-dev \
   libprotobuf-dev \
   libprotobuf-c0-dev \
   libseccomp-dev \
   libselinux1-dev \
+  libsystemd-dev \
   pkg-config \
+  runc \
   uidmap
 ```
 
-Debian, Ubuntu, and related distributions will also need to do the following setup:
+### Building missing dependencies
+
+If any dependencies cannot be installed or are not sufficiently current, they have to be built from source.
+This will mainly affect Debian, Ubuntu, and related distributions, or RHEL where no subscription is active (e.g. Cloud VMs).
+
+#### ostree
+
+A copy of the development libraries for `ostree` is necessary, either in the form of the `libostree-dev` package
+from the [flatpak](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak) PPA,
+or built [from source](https://github.com/ostreedev/ostree/blob/master/docs/contributing-tutorial.md)
+(see also [here](https://ostree.readthedocs.io/en/latest/#building)). As of Ubuntu 18.04, `libostree-dev` is available in the main repositories,
+and the PPA is no longer required.
+
+To build, use the following (running `make` can take a while):
+```bash
+git clone https://github.com/ostreedev/ostree ~/ostree
+cd ~/ostree
+git submodule update --init
+# for Fedora, CentOS, RHEL
+sudo yum install -y automake bison e2fsprogs-devel fuse-devel libtool xz-devel zlib-devel
+# for Debian, Ubuntu etc.
+sudo apt-get install -y automake bison e2fsprogs fuse liblzma-dev libtool zlib1g
+
+./autogen.sh --prefix=/usr --libdir=/usr/lib64 --sysconfdir=/etc
+# remove --nonet option due to https:/github.com/ostreedev/ostree/issues/1374
+sed -i '/.*--nonet.*/d' ./Makefile-man.am
+make
+sudo make install
+```
+
+#### golang
+
+Be careful to double-check that the version of golang is new enough (i.e. `go version`), version 1.10.x or higher is required.
+If needed, golang kits are available at https://golang.org/dl/. Alternatively, go can be built from source as follows
+(it's helpful to leave the system-go installed, to avoid having to [bootstrap go](https://golang.org/doc/install/source):
+
+```bash
+export GOPATH=~/go
+git clone https://go.googlesource.com/go $GOPATH
+cd $GOPATH
+git checkout tags/go1.10.8  # optional
+cd src
+./all.bash
+export PATH=$GOPATH/bin:$PATH
+```
+
+#### conmon
+
+The latest version of `conmon` is expected to be installed on the system. Conmon is used to monitor OCI Runtimes.
+To build from source, use the following (if not already executed above, run `export GOPATH=~/go && mkdir -p $GOPATH`):
```

Comment:
Text is almost verbatim from above, install instructions verbatim from the tutorial, except that the location of the git repo has changed from `github.com/kubernetes-sigs/cri-o` to `github.com/cri-o/cri-o`.